multicluster/apis/multicluster/v1alpha1/multiclusterset_types.go

```diff
@@ -23,19 +23,69 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+type ClusterStatus string
+
+const (
+	ClusterStatusReachable   ClusterStatus = "Reachable"
+	ClusterStatusUnreachable ClusterStatus = "Unreachable"
+)
+
+// MemberCluster defines member cluster information.
+type MemberCluster struct {
+	// Identify member cluster in multi-cluster set.
+	ClusterID string `json:"clusterID,omitempty"`
+	// API server of the member or the leader cluster.
+	Server string `json:"server,omitempty"`
+	// Secret name to access API server of the member from the leader cluster.
+	Secret string `json:"secret,omitempty"`
+	// ServiceAccount used by the member cluster to access into leader cluster.
+	ServiceAccount string `json:"serviceAccount,omitempty"`
+}
+
+// MemberClusterStatus defines member cluster status in the leader clusters.
+type MemberClusterStatus struct {
+	// ClusterID identifies member cluster in the set.
+	ClusterID string `json:"clusterID,omitempty"`
+	// IsLeader is true if the member cluster selected local .
```

Comment:
Im not sure what member cluster selected local means? seems like the comment is not complete?

Reply:
done.